native/python_package/test/test_alpine.sh

```diff
@@ -1,2 +1,5 @@
 #!/bin/sh
-time sh -c './sendmsg.py /usr/lib/python2.7/*.py | docker run -i --name python_driver --rm python_driver_alpine:latest > /dev/null'
+# Alpine 3.6:
+time sh -c './sendmsg.py /usr/lib/python2.7/*.py | docker run -i --name python_driver --rm 1edcc008c3f6 /opt/driver/bin/native > /dev/null'
+# python-alpine (3.5)
+#time sh -c './sendmsg.py /usr/lib/python2.7/*.py | docker run -i --name python_driver --rm 04d2a815082e /opt/driver/bin/native > /dev/null' 
```

Comment:
Do not leave dead code in comments, this just accumulates over time and you have the code in the git history.